core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournal.java

```diff
@@ -146,7 +146,8 @@ protected static UnderFileSystemConfiguration getJournalUfsConf() {
    */
   public UfsJournal(URI location, Master master, long quietPeriodMs,
       Supplier<Set<JournalSink>> journalSinks) {
-    this(location, master, UnderFileSystem.Factory.create(location.toString(), getJournalUfsConf()),
+    this(location, master, master.getMasterContext().getUfsManager().getRoot().acquireUfsResource()
```

Comment:
The root UFS is different from the journal UFS. If the UfsManager doesn't have it already, it should have a method for acquiring the journal UFS (to also use the journal conf, instead of the root ufs conf).

You can basically think of the journal UFS as a different mount point (separate from any other mount point), which is just for accessing the journal UFS.

Reply:
OK